backend/infrahub/workflows/catalogue.py

```diff
@@ -340,13 +340,42 @@
     function="repository_checks",
 )
 
+WEBHOOK_CONFIGURE = WorkflowDefinition(
+    name="webhook-setup-automations",
+    type=WorkflowType.USER,
+    module="infrahub.webhook.tasks",
+    function="configure_webhooks",
+)
+
+WEBHOOK_TRIGGER = WorkflowDefinition(
+    name="webhook-trigger-actions",
+    type=WorkflowType.USER,
+    module="infrahub.webhook.tasks",
+    function="trigger_webhooks",
+)
+
+WEBHOOK_TRIGGER = WorkflowDefinition(
```

Comment:
WEBHOOK_TRIGGER is defined twice

Reply:
Ah, yes thanks. I've removed the duplicate.